api/queries_repo.go

```diff
@@ -741,7 +741,7 @@ func RepoMetadata(client *Client, repo ghrepo.Interface, input RepoMetadataInput
 		go func() {
 			teams, err := OrganizationTeams(client, repo)
 			// TODO: better detection of non-org repos
-			if err != nil && !strings.HasPrefix(err.Error(), "Could not resolve to an Organization") {
+			if err != nil && !strings.Contains(err.Error(), "Could not resolve to an Organization") {
```

Comment:
This was already super fragile checking an error message string, luckily we had tests to catch this since the new library slightly changes the error message format. I did a quick search and I think these are the only two places that should be effected by that.